src/App.tsx

```diff
@@ -1,61 +1,123 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [hasTitleErorr, setHasTitleError] = useState(false);
```

Comment:
There is a typo in the variable name `hasTitleErorr`. It should be `hasTitleError` to maintain consistency and avoid potential bugs.